README.md

```diff
@@ -48,7 +48,7 @@ To install the frontend please do the following:
 ## Start
 To start the frontend please do the following:
 
-- Start the server by running `npm start` in the repo folder.
+- Start the server by running `npm start` in the repo folder. In some cases, that will not work so try `ember serve`.
```

Comment:
Saying "In some cases" is kind of confusing.  Maybe you could say something like, if npm start doesn't work you you, you can try ember serve as an alternative?